npm-search/assets/css/base.css

```diff
@@ -5,6 +5,11 @@ body {
   margin: 0; 
 } 
 
+h1 {
+  -webkit-margin-before: 0.3em;
```

Comment:
I added the <h1> tags on the page for SEO purposes. But the default margins above and below were huge and our page title didn't look good so I reduced it.